src/main/java/com/codecool/barbershop/barbershop/booking/BookingService.java

```diff
@@ -61,19 +61,12 @@ public Booking updateStatusBooking(BookingReqChangeStatus bookingRequest) {
         return bookingRepository.findAllByClient_ClientId(clientId,sort);
     }
 
-    public DashboardData getDataForDashBoard() {
-        DashboardData data = new DashboardData();
 
-        Date firstDayOfTheMonth = java.sql.Date.valueOf(LocalDate.now().with(TemporalAdjusters.firstDayOfMonth()));
-        Date lastDayOfTheMonth = java.sql.Date.valueOf(LocalDate.now().with(TemporalAdjusters.lastDayOfMonth()));
-
-        data.setTotalClients(clientService.getTotalClients());
-        data.setTotalConfirmedBookings(bookingRepository.countBookingModelByBookingDateBetweenAndBookingStatus(firstDayOfTheMonth, lastDayOfTheMonth, BookingStatus.CONFIRM));
-        data.setTotalUpcomingBookings(bookingRepository.countBookingModelByBookingDateBetweenAndBookingStatus(firstDayOfTheMonth, lastDayOfTheMonth, BookingStatus.UPCOMING));
-        data.setTotalCanceledBookings(bookingRepository.countBookingModelByBookingDateBetweenAndBookingStatus(firstDayOfTheMonth, lastDayOfTheMonth, BookingStatus.CANCEL));
-        data.setLatestBookings(bookingRepository.findTop9ByBookingStatusOrderByBookingDateAsc(BookingStatus.UPCOMING));
-
-        return data;
+    public int countBookingsByBookingDateBetweenAndBookingStatus(Date firstDayOfTheMonth, Date lastDayOfTheMonth, BookingStatus confirm) {
```

Comment:
Very nice refactoring.  
Here the magic strings were not so obvious, but you handled them like a pro -> parametrized the BookingStatus